compressors/integer_coders.py

```diff
@@ -2,6 +2,7 @@
 from core.data_encoder_decoder import DataEncoder, DataDecoder
```

Comment:
Wondering if it is better to just have one each compressor in a separate file. 
Probably that ll be better

Reply:
Thanks, will separate it out into another file